### PR TITLE
fix(SRVKP-6610): correct link to SRE docs and perform cleanup

### PR DESCRIPTION
**Pull request by divyansh42**
This PR changes the runbook_url after the SRE docs PR is merged.

Signed-off-by: Divyanshu Agrawal [diagrawa@redhat.com](mailto:diagrawa@redhat.com)

/cc @mftb 
FYI @gabemontero

### CODE REVIEW

**Comment by divyansh42** [comment]
@mftb could you please review this PR?

**Comment by mftb** [comment]
@divyansh42 looks like you have a couple of issues. The first one is that gitlint did not like your commit message:
`1: CT1 Title does not follow ConventionalCommits.org format 'type(optional-scope): description': "Change link to the SRE docs after the relevant PR merge"`

Also, looks like a prometheus rule is failing its test:
`time="2024-10-21T13:22:40Z" level=fatal msg="[test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml] failed to run 'promtool test rules'; output:\nUnit Testing:  /tmp/obsctl-reloader-rules-checker2404177132/pipeline_deadlock_crashloop_test.yaml\n  FAILED:\n    alertname: PipelineControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"PipelineControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton pipeline controller on cluster cluster01 has appeared deadlocked on 2 pipelineruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton pipeline controller appears to have stopped processing active pipelineruns which have not been started yet.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"PipelineControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton pipeline controller on cluster cluster01 has appeared deadlocked on 2 pipelineruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton pipeline controller appears to have stopped processing active pipelineruns which have not been started yet.\", team=\"pipelines\"}\n            ]\n\n    alertname: TaskControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"TaskControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton taskrun controller on cluster cluster01 has appeared deadlocked on 2 taskruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton taskrun controller appears to have stopped processing active taskruns whose underlying Pod have not failed Kubernetes screening.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"TaskControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton taskrun controller on cluster cluster01 has appeared deadlocked on 2 taskruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton taskrun controller appears to have stopped processing active taskruns whose underlying Pod have not failed Kubernetes screening.\", team=\"pipelines\"}\n            ]\n\n    alertname: TaskControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"TaskControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton taskrun controller on cluster cluster01 has appeared deadlocked on 2 taskruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton taskrun controller appears to have stopped processing active taskruns whose underlying Pod have not failed Kubernetes screening.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"TaskControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton taskrun controller on cluster cluster01 has appeared deadlocked on 2 taskruns.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton taskrun controller appears to have stopped processing active taskruns whose underlying Pod have not failed Kubernetes screening.\", team=\"pipelines\"}\n            ]\n\n    alertname: ResolverControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"ResolverControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton resolver controller on cluster cluster01 has appeared deadlocked on 2 resolutionrequests.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton resolver controller appears to have stopped processing active resolutionrequests which have not been started yet.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"ResolverControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton resolver controller on cluster cluster01 has appeared deadlocked on 2 resolutionrequests.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton resolver controller appears to have stopped processing active resolutionrequests which have not been started yet.\", team=\"pipelines\"}\n            ]\n\n    alertname: ChainsControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"ChainsControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton chains controller on cluster cluster01 has appeared deadlocked on 600 pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton chains controller appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"ChainsControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton chains controller on cluster cluster01 has appeared deadlocked on 600 pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton chains controller appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ]\n\n    alertname: ResultsDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"ResultsDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton results on cluster cluster01 appears deadlocked on pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton results appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"ResultsDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton results on cluster cluster01 appears deadlocked on pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton results appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ]\n\n    alertname: PACControllerDeadlock, time: 1h25m, \n        exp:[\n            0:\n              Labels:{alertname=\"PACControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton PAC controller on cluster cluster01 has appeared deadlocked on 600 pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton PAC controller appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ], \n        got:[\n            0:\n              Labels:{alertname=\"PACControllerDeadlock\", severity=\"critical\", slo=\"true\", source_cluster=\"cluster01\"}\n              Annotations:{alert_team_handle=\"<!subteam^S04PYECHCCU>\", description=\"Tekton PAC controller on cluster cluster01 has appeared deadlocked on 600 pipelinerun events.\", runbook_url=\"https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-pipeline-related-deadlocks.md\", summary=\"Tekton PAC controller appears to have stopped processing active pipelineruns.\", team=\"pipelines\"}\n            ]\n\n\n\n"
`

Please fix both

**Comment by divyansh42** [comment]
Thanks @mftb for taking a look. I am not able to see the errors in the console. Do I need to have some special permissions for this?

**Comment by mftb** [comment]
> Thanks @mftb for taking a look. I am not able to see the errors in the console. Do I need to have some special permissions for this?

The logs disappear after a while, what I suggest you to do is to check them right after the pipeline run is triggered. If you need to check older logs, I think you have to search for them in Tekton Results

**Comment by divyansh42** [comment]
@mftb as soon as I open the PipelineRun logs link available in the test. It points me to "Join the Waitlist" page
image

**Comment by mftb** [comment]
You have to request access to the Konflux workspace there. Please click "join the waitlist" and tell me what happens

File: rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml [hunk 1]
```diff
@@ -126,7 +121,6 @@ spec:
               Tekton results on cluster {{ $labels.source_cluster }} only has an API success rate of {{ $value | humanizePercentage }}.
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
-            # link will be available once https://issues.redhat.com/browse/SRVKP-5900 is done
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-result-high-errors.md
```
Comment (mftb) [inline rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:124]: Should this still be the old link?

Comment (divyansh42) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:124]: No, this is the correct link. We don't need to change this.

**Review by mftb — APPROVED** [review]
/lgtm
/approve

**Pull request merged**